index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body class="container">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">Thank you for taking the time to help us improve the platform</p>
+      
+      <form id="survey-form">
+        <div class="input-container">
+          <div class="input-description">
+            <label id="name-label" for="name">Name</label>
+            <input required id="name" type="text" placeholder="Enter your name">
+          </div>
+          <div class="input-description">
+            <label id="email-label" for="email">Email</label>
+            <input required id="email" pattern=".+@globex\.com" type="email" placeholder="Enter your email">
+          </div>
+          <div class="input-description">
+            <label id="number-label" for="number">Age</label>
+            <input required id="number" type="number" placeholder="Enter a number" min="10" max="100">
+          </div>          
+        </div>        
+
+        <p class="center">Which option best describes your current role?</p>
+        <div class="selectContainer">
+          <select id="dropdown" name="" id="">
+            <option selected disabled value="">Select current role</option>
+            <option value="">Student</option>
+            <option value="">Full time job</option>
+            <option value="">Full time lerner</option>
+            <option value="">Preder not to say</option>
+          </select>
+        </div>
+        
+
+
+        <p class="center"> Would you recommend freeCodeCamp to a friend?</p>
+        <div class="radioButton">
+            <div>
+              <input id="definitely" type="radio" name="attribute" value="definitely">
+              <label for="definitely">Definitely</label>
+            </div>
+            <div>
+              <input id="maybe" type="radio" name="attribute" value="maybe">
+              <label for="maybe">Maybe</label>
+            </div>
+            <div>
+              <input id="notsure" type="radio" name="attribute" value="notsure"> 
+              <label for="notsure">Not sure</label>
+            </div>
```

Comment:
I think it would be better to use semantic tags.
Change the div tag to section tag

Reply:
- [ ] This is a really good recommendation but in this situation, I think you could explain what you mean by "semantic tags" or explain "why" you are asking for this change. Explaining "why" makes the code owner feel that is learning something new and it also makes the suggestion feel more friendly :wink:
- [ ] Also, "Change the div tag to section tag" still feels like an order. I would suggest rephrasing your comment to make it more friendly or you could try adding emojis :smiley: